subworkflows/local/prepare_intervals.nf

```diff
@@ -20,67 +20,96 @@ workflow PREPARE_INTERVALS {
 
     ch_versions = Channel.empty()
 
-    // TODO maybe instead []
     ch_intervals                     = Channel.empty()
     ch_intervals_bed_gz_tbi          = Channel.empty()
     ch_intervals_combined_bed_gz_tbi = Channel.empty() // Create bed.gz and bed.gz.tbi for input/or created interval file. Contains ALL regions.
-    tabix_in_combined = Channel.empty()
 
     if (params.no_intervals) {
         file("${params.outdir}/no_intervals.bed").text = "no_intervals\n"
+        file("${params.outdir}/no_intervals.bed.gz").text = "no_intervals\n"
+        file("${params.outdir}/no_intervals.bed.gz.tbi").text = "no_intervals\n"
+
         ch_intervals = Channel.fromPath(file("${params.outdir}/no_intervals.bed"))
-        tabix_in_combined = ch_intervals.map{it -> [[id:it.getName()], it] }
+                                            .map{ it -> [it, 0]}
+
+        ch_intervals_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
+                                            .collect().map{ it -> [it, 0]}
+
+        ch_intervals_combined_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
+                                            .collect()
```

Comment:
Yes otherwise their are not one element, if that makes sense see:
```
with collect(): [/Users/monarchy/Projects/Coding/sarek/results/no_intervals.bed.gz.tbi, /Users/monarchy/Projects/Coding/sarek/results/no_intervals.bed.gz]
without collect():
/Users/monarchy/Projects/Coding/sarek/results/no_intervals.bed.gz.tbi
/Users/monarchy/Projects/Coding/sarek/results/no_intervals.bed.gz
```